data/textinfo

```diff
@@ -13,10 +13,10 @@ ip -6 r s
 cat /etc/resolv.conf
 ls -al /etc/sysconfig/network
 rpm -qa 'kernel-*'
-grep DISPLAYMANAGER /etc/sysconfig/displaymanager
-grep DEFAULT /etc/sysconfig/windowmanager
-ls -l /etc/ntp*
-[ -f /var/log/messages ] && du /var/log/messages
+grep DISPLAYMANAGER /etc/sysconfig/displaymanager || true
+grep DEFAULT /etc/sysconfig/windowmanager || true
+ls -l /etc/ntp* || true
+du /var/log/messages || true
```

Comment:
why do you replace my check for the file existance with just calling du on a potentially non-existing file and then ignoring the exit status?